src/components/fields/money-field/money-field.js

```diff
@@ -107,7 +111,10 @@ class MoneyField extends React.Component {
             badge={
               this.props.hasHighPrecisionBadge &&
               !MoneyInput.isEmpty(this.props.value) &&
-              MoneyInput.isHighPrecision(this.props.value) ? (
+              MoneyInput.isHighPrecision(
+                this.props.value,
+                this.props.intl.locale
```

Comment:
I'm still confused about this. Why do we use this locale to determine this? I don't need to have my computer set to Japanese to enter prices in Yen

Reply:
couldn't we just determine with the currency code whether it's capable of being high precision?

Reply:
As far as I know, the `intl.locale` is what you pass to the `IntlProvider`, which is usually what we set.

Reply:
> Why do we use this locale to determine this?

In currencies like JPY the number `12.555` is high precision according to the `en` local, but cent precision according to the `de` locale.

> I don't need to have my computer set to Japanese to enter prices in Yen

You don't, you can enter the number in all cases, but whether it is  high precision or not should be based on the locale

> couldn't we just determine with the currency code whether it's capable of being high precision

All currencies can be high precision if the entered value has more fractions than the currency's `fractionDigits`

> As far as I know, the `intl.locale` is what you pass to the `IntlProvider`, which is usually what we set

Yes it is, would it be better if we get a prob from the user instead?

Reply:
I thought in currencies like JPY there is no such thing as cent precision or high precision in any locales. 

There's no such thing as cents, so how could they have high precision prices? 😕

Reply:
When the price of example was 7 JPY and there is 50% off then it's 3.5 JPY.
there's no cents or smaller units, but there are the cases when u have a fractions of the smallest unit.

Reply:
it's like Euro, it has 2 fraction digits `.01` is cent, but u can have a balance of `1.0000003` euro

Reply:
Yen doesn't support fractional fractional units though. You can't have 3.5 JPY 😕 Can you find a source supporting this fact? AFAIK they removed the "sen" value from their currency in 1950s.

Reply:
https://www.xe.com/currencyconverter/convert/?Amount=1&From=EUR&To=JPY
they removed the "sen", but you can still have .3 JPY

Reply:
Can we involve someone from the API in this discussion and find out whether or not the API actually supports fractional YEN amounts?

Reply:
I believe that fractional digits are only used for exchange and stock trading

Reply:
Any currency can be high precision and it should not depend on any locale from my understanding. The use case are product variants with very low unit prices e.g print items like business cards, envelops etc. The price for one single business card could be EUR 0,00001 ... For EUR any price with more than 2 fraction digits is high precision. For JPY, any price with => 1 fraction digit is high precision. Lets do a short meeting tomorrow all together to get rid of the confusion